crates/miden-lib/src/note/scripts.rs

```diff
@@ -33,8 +33,7 @@ pub fn p2id() -> NoteScript {
 
 /// Returns the P2ID (Pay-to-ID) note script commitment.
 pub fn p2id_commitment() -> Digest {
-    let bytes = include_bytes!(concat!(env!("OUT_DIR"), "/assets/note_scripts/P2ID_commitment"));
-    Digest::try_from(bytes).unwrap()
+    P2ID_SCRIPT.clone().hash()
```

Comment:
```suggestion
    P2ID_SCRIPT.hash()
```

I think this should work as well.